core/nuxeo-notify-behavior.js

```diff
@@ -0,0 +1,42 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import '@polymer/polymer/polymer-legacy.js';
+
+let fallback;
+
+/**
+ * @polymerBehavior Nuxeo.NotifyBehavior
+ */
+export const NotifyBehavior = {
+  notify(options) {
+    const target = this.isConnected ? this : fallback;
+    if (!target) {
+      return;
+    }
+    target.dispatchEvent(
+      new CustomEvent('notify', {
+        composed: true,
+        bubbles: true,
+        detail: options,
+      }),
+    );
+  },
+
+  setFallbackNotificationTarget(el = this) {
```

Comment:
I guess this method would be better as global to make it easier to use when not using Polymer, similar to the `setI18n` and `setRouter` which were introduced recently for that purpose too

Reply:
I agree. And I think it's safer too.